app/js/views/tag-item.jsx

```diff
@@ -13,28 +13,7 @@ export default class TagItem extends React.Component {
   }
 
   handleSetTag(evt) {
-    const checked = evt.target.checked;
```

Comment:
It's not properly supported right now so it just introduces maintenance burden, _but_ supporting tags is the next thing I'm going to do after watchers patch.